docs/extend/legacy_plugins.md

```diff
@@ -88,7 +88,7 @@ Plugin
  Plugin                                                       | Description
 ------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 [Casbin AuthZ Plugin](https://github.com/casbin/casbin-authz-plugin) | An authorization plugin based on [Casbin](https://github.com/casbin/casbin), which supports access control models like ACL, RBAC, ABAC. The access control model can be customized. The policy can be persisted into file or DB.
-[HBM plugin](https://github.com/kassisol/hbm)                 | An authorization plugin that prevents from executing commands with certains parameters.
+[HBM plugin](https://github.com/kassisol/hbm)                 | HBM (Harbormaster) is an authorization plugin that manages usage of certains commands or parameters passed to Docker daemon. For example, control which images can be pulled or start containers with certains parameters such as: --privileged, --{ipc,net,pid,userns,uts}=host, any Linux capabilities with parameter --cap-add, any devices added with parameter --device, any dns servers added with parameter --dns, any ports added with parameter --port, any volumes mounted with parameter -v, any logging with parameters "--log-driver" and "--log-opt".
```

Comment:
The "for example ..." is probably a bit too much information, and best left to the plugin's documentation